### PR TITLE
Added remove spaces from a string

### PR DESCRIPTION
**Pull request by RaiyanMahin**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

For string related problems we need to remove spaces to get our expected output , that can be done by getline function in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#2604

### CODE REVIEW

File: C-Plus-Plus/cp/remove_spaces_from_a_string.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    // and getting every word
+    while (getline(s2, tem, ' ')) {
+       s += tem;
+    }
+    return s;
+}
+
+int main()
+{
+    string s;
```
Comment (atarax665) [inline C-Plus-Plus/cp/remove_spaces_from_a_string.cpp:26]: Please format input/output to make your program more interactive.

File: C-Plus-Plus/cp/remove_spaces_from_a_string.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// Given a string , remove spaces from it and then return new string
```
Comment (atarax665) [inline C-Plus-Plus/cp/remove_spaces_from_a_string.cpp]: Add both the problem description and your approach in a single multi-line comment.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
I have resolved the changes.
Kindly review.

**Comment by RaiyanMahin** [comment]
@atarax665 kindly review.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**